tf-dataflow/real_data_loader.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/python3
+# https://github.com/tensorflow/tpu/blob/master/tools/datasets/jpeg_to_tf_record.py
+# https://www.tensorflow.org/tutorials/load_data/tfrecord
+
+from __future__ import print_function
+
+import _thread
+import glob
+import json
+import logging
+import os
+import random
+import sys
+
+import tensorflow.compat.v1 as tf
+
+tf.disable_eager_execution()
+
+LOG = logging.getLogger()
+logging.basicConfig(level=logging.DEBUG)
+LOG.addHandler(logging.StreamHandler())
+
+
+_train_val_ratio = 0.8
```

Comment:
Python constants should be ALL CAPS according to PEP8 standard.

Reply:
fixed